server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableUtils.java

```diff
@@ -303,9 +312,11 @@ public TableId getTableId() {
 
     Selection selection = selector.select(new CompactionSelector.SelectionParameters() {
 
+      private final ServiceEnvironment senv = new ServiceEnvironmentImpl(tablet.getContext());
+
```

Comment:
This is duplicated in the `selectFiles` method for InitParamaters and for SelectionParameters. It could be created once in the selectFiles method as a final local variable.